docs/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Dashboard's auto-update feature works with multiple Lifecycle replicas.
+  Live updates can now be enabled on the jobs list. It is turned off by default.
```

Comment:
Is there a reason we change default behaviours? To me, automatic updates seem like the type of thing you'd want on by default.

Reply:
Well, in the past this feature generated a lot of requests, because clients were stuck in a reconnection loop and trying to get the fresh data from a server. That's why I'm a bit more careful this time. I don't want to strain the servers too much. Let's see how it goes and we can make it default again in the future.
The good news is that this option is saved in a local storage of a browser, so if you set it once, it will be turned on even after refreshing or closing the page.